terraform/modules/tf_stream_alert/iam.tf

```diff
@@ -234,7 +234,7 @@ data "aws_iam_policy_document" "alert_processor_firehose" {
     ]
 
     resources = [
-      "${aws_kinesis_firehose_delivery_stream.stream_alerts.arn}",
+      "arn:aws:firehose:${var.region}:${var.account_id}:deliverystream/${var.prefix}_streamalert_alert_delivery",
```

Comment:
Any reason to use full arn of firehose delivery stream, instead of `${aws_kinesis_firehose_delivery_stream.stream_alerts.arn}`?

Reply:
That resource isn't in this module, so that interpolation isn't possible.  I'd have to pass it in as a variable to this module instead, which I can do!

Reply:
You are right. Thanks for the explanation.

Reply:
I'm actually going to leave it, since we do the same thing for Athena data streams